liquid_node/configuration.py

```diff
@@ -160,6 +160,8 @@ def __init__(self):
         self.tika_count = self.ini.getint('liquid', 'tika_count', fallback=1)
         self.tika_memory_limit = self.ini.getint('liquid', 'tika_memory_limit', fallback=800)
 
+        self.thumbnail_memory_limit = self.ini.getint('liquid', 'thumbnail_memory_limit', fallback=800)
```

Comment:
is it really filling up 800MB of ram?

Take a look at the container RAM usage like this:
- visit http://10.66.60.1:9990/grafana
- visit the "docker stats" dashboard and look for the memory graph
- click your container name on the list
- observe max RAM usage while you re-process testdata with it